docs/src/tutorial/CollisionHandling.md

```diff
@@ -36,6 +36,12 @@ plot(bouncingSphere, "free.r", figure=1)
 
 end
 ```
+or
+```julia
+julia> import Modia3D
```

Comment:
not necessary?

Reply:
Yes and no. If users are only interested in this specific example, and haven't tried something else in their actual Julia session. They need it, otherwise of course not.
I think it is the best, that each example works on it's own, without any prerequisite.

Reply:
This will always work. There is no need to install an IDE. Okay, they could copy it into the Julia REPL as well :)

Reply:
Another point: Maybe we'll change a tutorial example in the repository, but we'll forget to adapt it in the docu. This should still work.

Reply:
'import Modia3D' is only required for '$(Modia3D.path)' in the include command. But for the user it looks as if it is necessary to run the model script. Imho this is rather misleading and I would prefer to go without it, i.e. 'julia> include("<Modia3D_path>/test/Tutorial/BouncingSphere.jl")'